input/fsh/codeSystem/codeSystem.fsh

```diff
@@ -0,0 +1,1319 @@
+* #9860213 ^designation[=].value = "Se identifica como género masculino"
+* #9860213 ^designation[+].language = #es
+* #9860213 ^designation[=].value = "Identidad de género masculino"
+* #9860213 ^designation[+].language = #es
+* #9860213 ^designation[=].value = "Identidad masculina"
+* #9860213 ^property[0].code = #conceptclass
+* #9860213 ^property[=].valueString = "Finding"
+* #9860213 ^property[+].code = #datatype
+* #9860213 ^property[=].valueString = "N/A"
+* #9860215 "Identifies as female gender"
+* #9860215 ^designation[0].language = #en
+* #9860215 ^designation[=].value = "Female"
+* #9860215 ^designation[=].use.code = #SHORT
+* #9860215 ^designation[+].language = #en
+* #9860215 ^designation[=].value = "Identifies as female gender"
+* #9860215 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860215 ^designation[+].language = #es
+* #9860215 ^designation[=].value = "Se identifica como género femenino"
+* #9860215 ^designation[+].language = #es
+* #9860215 ^designation[=].value = "Identidad de género femenino"
+* #9860215 ^designation[+].language = #es
+* #9860215 ^designation[=].value = "Identidad femenina"
+* #9860215 ^property[0].code = #conceptclass
+* #9860215 ^property[=].valueString = "Finding"
+* #9860215 ^property[+].code = #datatype
+* #9860215 ^property[=].valueString = "N/A"
+* #9860217 "Male to female transsexual"
+* #9860217 ^designation[0].language = #en
+* #9860217 ^designation[=].value = "Male to female transsexual"
+* #9860217 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860217 ^designation[+].language = #es
+* #9860217 ^designation[=].value = "Transexual, mujer a hombre"
+* #9860217 ^property[0].code = #conceptclass
+* #9860217 ^property[=].valueString = "Finding"
+* #9860217 ^property[+].code = #datatype
+* #9860217 ^property[=].valueString = "N/A"
+* #9860219 "Female to male transsexual"
+* #9860219 ^designation[0].language = #en
+* #9860219 ^designation[=].value = "Female to male transsexual"
+* #9860219 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860219 ^designation[+].language = #es
+* #9860219 ^designation[=].value = "Cirugía intersexual, conversión de sexo femenino a sexo masculino"
+* #9860219 ^property[0].code = #conceptclass
+* #9860219 ^property[=].valueString = "Finding"
+* #9860219 ^property[+].code = #datatype
+* #9860219 ^property[=].valueString = "N/A"
+* #9860221 "Identifies as non-binary gender"
+* #9860221 ^designation[0].language = #en
+* #9860221 ^designation[=].value = "Identifies as non-binary gender"
+* #9860221 ^designation[=].use.code = #FULLY_SPECIFIED
+* #9860221 ^designation[+].language = #es
+* #9860221 ^designation[=].value = "Se identifica como género no binario"
+* #9860221 ^designation[+].language = #es
+* #9860221 ^designation[=].value = "Identidad de género no binario"
+* #9860221 ^property[0].code = #conceptclass
+* #9860221 ^property[=].valueString = "Finding"
+* #9860221 ^property[+].code = #datatype
+* #9860221 ^property[=].valueString = "N/A"
+* #9860223 "Female gender"
```

Comment:
Noting that there are two codes related to Female Gender - #9860223 and #9860215

Reply:
Fixed and removed the, #9860215 concept.